src/internet/radiobrowser/radiobrowserservice.cpp

```diff
@@ -419,9 +420,28 @@ void RadioBrowserService::ResolveStationUrl(const QUrl& original_url) {
     reply->deleteLater();
     QJsonDocument document = ParseJsonReply(reply);
 
-    QJsonObject item = document.object();
+    StreamList list;
+    QJsonArray contents = document.array();
+    QJsonValue item = contents.first();
+    if (item.isUndefined()) return;
+
+    Song ret;
+    ret.set_valid(true);
+    ret.set_title(item["name"].toString());
```

Comment:
You're build is failing on some platforms because this operator[] override was added in Qt 5.10, but Stretch is still on 5.7.

Reply:
Fixed, thank you